src/main/kotlin/org/rust/ide/utils/import/ImportCandidatesCollector.kt

```diff
@@ -61,6 +61,14 @@ object ImportCandidatesCollector {
             // check that result after import can be resolved and resolved element is suitable
             // if no, don't add it in candidate list
             .filter { canBeResolvedToSuitableItem(importingPathText, importContext, it.info) }
+            // make std appear first
+            .sortedBy {
+                if (it.qualifiedNamedItem.containingCrate?.origin == PackageOrigin.STDLIB) {
```

Comment:
What's about code from your workspace?
If we want to make any sorting here, we want to put items from workspace first